src/curves/ed25519.ts

```diff
@@ -0,0 +1,19 @@
+import { getPublicKey as getEd25519PublicKey } from '@noble/ed25519';
+
+export { CURVE as curve } from '@noble/ed25519';
+
+export const name = 'ed25519';
+export const secret = Buffer.from('ed25519 seed', 'utf8');
```

Comment:
I find it weird that the secrets are hard-coded. Shouldn't be they customisable and provided by the extension?

Reply:
No, these are [defined by SLIP-10 (and BIP-32)](https://github.com/satoshilabs/slips/blob/master/slip-0010.md#master-key-generation).